bin/checks.ts

```
@@ -0,0 +1,159 @@
+#!/usr/bin/env node
+/* eslint-disable @typescript-eslint/camelcase */
+
+import args from 'args'
+
+import { printSummary } from '../src/checks/checks-common'
+import { eslintCheck, EslintInput } from '../src/checks/eslint/eslint'
+import { runEslint } from '../src/checks/eslint/run-eslint'
+import { jestCheck, JestInput } from '../src/checks/jest/jest'
+import { runJest, runReactScriptsTest } from '../src/checks/jest/run-jest'
+import { mochaCheck, MochaInput } from '../src/checks/mocha/mocha'
+import { runMocha } from '../src/checks/mocha/run-mocha'
+import { auditCheck, AuditInput } from '../src/checks/npm-audit/audit'
+import { runNpmAudit } from '../src/checks/npm-audit/run-audit'
+import { runTsc } from '../src/checks/tsc/run-tsc'
+import { tscCheck } from '../src/checks/tsc/tsc'
+const { REPO_NAME, COMMIT_SHA } = process.env
```

Maybe fail if they are not set as you need the for posting to github